### ISSUE TITLE
Ensure screenreaders announce the page title / change when reaching the confirmation page

### ISSUE THREAD

**Issue by brdunfield** [cds-snc/platform-forms-client#319]
When a form is submitted while using a screenreader, the screenreader will correctly read out the "we are processing your form" blurb while the form is submitting. However, once the confirmation page appears, the screenreader does not announce the new page title or heading.

This should be improved to give folks using these assistive technologies the same confirmation that their form has submitted, without them having to know that they need to navigate through a new page to figure out whether the form has succeeded or not.

**Comment by srtalbot** [comment]
Example form: https://forms-staging.cdssandbox.xyz/id/77 

Acceptance criteria:
- [ ] Focus state starts at the confirmation of the submission (e.g., "Thank you for your submission")

**Comment by craigzour** [comment]
Investigation results so far:
- When a form has been submitted a new page is generated so the screen reader reads it from the beginning. We pretty much have the same issue when the form page is being presented to the user.
- The reason why the "form submission is in progress" message is being read by the screen reader is because the page stays but the content is dynamically updated and we have implemented the `role="status"` property (https://github.com/cds-snc/platform-forms-client/blob/main/components/globals/Loader.tsx#L21) which is used to tell the screen reader that a new element has been added to the page and it needs to be read.

Here is a solution I quickly tested:
- In DynamicForm.tsx I added a piece of code that automatically focuses the first `h1` HTML element (which is the "Form has been submitted" message in our case).

```
useEffect(() => {
  const element = document.getElementsByTagName("h1").item(0);
  element?.focus();
}, []);
```

**Comment by srtalbot** [comment]
From a flow perspective that seems like it'll work. The H1s will remain the main "thank you" content on the exit page.
